WFP-VAM/prism-frontend
# Use selected date as end date for popup charts

### Description

This fixes https://github.com/WFP-VAM/prism-app/issues/1008.

Use selected date as the end date for popup charts

## How to test the feature:

- Load 10-day aggregate rainfall layer
- Change date to any historical date
- Click on an admin area and then click on the chart tooltip
- See the date in the timeline match the end date of the chart

## Checklist - did you ...



Test your changes with


## Screenshot/video of feature:
<img width="1512" alt="Screenshot 2023-12-06 at 20 04 37" src="https://github.com/WFP-VAM/prism-app/assets/3343536/2f6fda18-5271-4e79-bfb9-835a860f3d1d">
